src/test/java/org/broadinstitute/hellbender/IntelDeflaterTest.java

```diff
@@ -0,0 +1,24 @@
+package org.broadinstitute.hellbender;
+
+import htsjdk.samtools.util.zip.DeflaterFactory;
+import org.apache.commons.lang3.SystemUtils;
+import org.testng.Assert;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+/**
+ * Test that it's possible to load libIntelDeflater
+ */
+public class IntelDeflaterTest {
```

Comment:
It's not really a unit test, I guess it's sort of an integration test?  What suffix do you prefer?